jenkins/internal-ci/generic/pr/cortx_deployment/motr.groovy

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env groovy
+// CLEANUP REQUIRED
+pipeline { 
+    agent {
+        node {
+            label "docker-${OS_VERSION}-node"
+        }
+    }
+
+    options { 
+        skipDefaultCheckout()
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        disableConcurrentBuilds()
+        ansiColor('xterm')  
+    }
+
+    parameters {  
+	    string(name: 'MOTR_URL', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Repo for Motr')
+        string(name: 'MOTR_BRANCH', defaultValue: 'main', description: 'Branch for Motr')
+        choice(name: 'DEPLOY_BUILD_ON_NODES', choices: ["Both", "1node", "3node" ], description: '''<pre>If you select Both then build will be deploy on 1 node as well as 3 node. If you select 1 node then build will be deploy on 1 node only. If you select 3 node then build will be deploy on 3 node only. 
+</pre>''')
+        string(name: 'CORTX_RE_URL', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repo for cortx-re')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch for cortx-re')
+
+        choice (
+            choices: ['all', 'cortx-all' , 'cortx-rgw', 'cortx-data'],
```

Comment:
Remove `cortx-all` and add `cortx-control`

Reply:
Done